views/shared/profiles/achievements.jade

```diff
@@ -11,6 +11,12 @@
 // <div class='achievement achievement-shield'></div>
 // <div class='achievement achievement-karaoke'></div>
 
+mixin aLink(url, label)
```

Comment:
Seems like this would be useful to have access to for all pages. Maybe put it here: https://github.com/HabitRPG/habitrpg/blob/develop/views/shared/mixins.jade

That gets included in the main index.jade file, so it's accessible to all templates. 

Though, you may need to include that mixin file in the mobile layout file as well.